index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>porofolio main page</title>
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://kit.fontawesome.com/091ffb7972.js" crossorigin="anonymous"></script>
+</head>
+<body>
+      <header class="head">
+        <nav>
+          <h2 class="pagename" >Welcome</h2>
+        </nav>
+        <img src="tools/Union.svg" alt="" class="nav-icon">
+    </header>
+    <section class="welcome">
+        <p ><span class="greetings">Hey there. I’m Josue</span><br>
+        <span class="profession">I’m a software developer</span></p>
```

Comment:
- [ ] Kindly use `h1` tag instead of `p` tag for your title to follow the html heading hierarchy 👍